shared/ceres.gradle

```diff
@@ -4,7 +4,6 @@ nativeUtils {
             groupId = "edu.wpi.first.thirdparty.frc2024.ceres"
             artifactId = "ceres-cpp"
             headerClassifier = "headers"
-            sourceClassifier = "sources"
```

Comment:
Unrelated change?

Reply:
Yes, but #7844 will likely be merged beforehand and make this OBE.

Reply:
Was the change missed in 7844?

Reply:
We haven't merged main into 2027 since then.